src/test/operators/jit_operator/operators/jit_compute_test.cpp

```diff
@@ -41,11 +43,17 @@ TEST_F(JitComputeTest, TriggersComputationOfNestedExpression) {
   compute->set_next_operator(sink);
 
   // We test the correct computation of the expression ten times with random values
+  std::random_device rd;
+  std::mt19937 gen(rd());
+
   for (auto i = 0; i < 10; ++i) {
     // Create input tuple values
-    auto a = static_cast<int32_t>(std::rand());
-    auto b = static_cast<int32_t>(std::rand());
-    auto c = static_cast<int32_t>(std::rand());
+    std::uniform_int_distribution<> dis(0, std::numeric_limits<int32_t>::max());
+    auto a = static_cast<int32_t>(dis(gen));
```

Comment:
Wouldn't `std::uniform_int_distribution<int32_t>` mean that you don't need this cast?

Reply:
Yes, that makes it clearer. Is changed.